modules/irsa/main.tf

```diff
@@ -46,7 +46,7 @@ resource "aws_iam_role" "irsa" {
 
   tags = merge(
     {
-      "Name" = format("%s-%s-%s", var.addon_context.eks_cluster_id, trim(var.kubernetes_service_account, "-*"), "irsa"),
+      "Name" = var.addon_context.irsa_iam_role_name != null ? var.addon_context.irsa_iam_role_name : format("%s-%s-%s", var.addon_context.eks_cluster_id, trim(var.kubernetes_service_account, "-*"), "irsa"),
```

Comment:
The role name is no longer controlled through tag `Name` like previously so we can just remove this tag